### ISSUE TITLE
Convert to the new ASB client

### ISSUE THREAD

**Issue by SeanFeldman** [MessageHandler/MessageHandler.Runtime.EventProcessing#3]
@yvesgoeleven let me know when this can start.

**Comment by yvesgoeleven** [comment]
@SeanFeldman Will split out the work this evening, but probably need to do some work on the nuget packages of the base runtime before we can proceed

**Comment by yvesgoeleven** [comment]
Closing this in favor of the other issues
